### PR TITLE
Fix Dutch locale for Moment.JS

### PR DESCRIPTION
**Pull request by daan944**
NL locale for Moment.JS was never imported, so some date/times were displayed in Portuguese (I guess since that's the last imported value).

To test: Set locale to Dutch, view a crud list admin panel and see Dutch as datetime. And: Open a comment thread, click a username and see the date in Dutch.

### CODE REVIEW

File: client/coral-framework/services/i18n.js [hunk 1]
```diff
@@ -18,21 +19,21 @@ import daTA from 'timeago.js/locales/da';
 import deTA from 'timeago.js/locales/de';
 import esTA from 'timeago.js/locales/es';
 import frTA from 'timeago.js/locales/fr';
+import nl from 'timeago.js/locales/nl';
```
Comment (kgardnr) [inline client/coral-framework/services/i18n.js]: Should this be `nlTA`?

Comment (daan944) [reply client/coral-framework/services/i18n.js]: I only sorted those entries. For consistency it might be better to rename it, yes.

Comment (kgardnr) [reply client/coral-framework/services/i18n.js]: Ok great can you change it and we will merge it? Thanks @daan944!

Comment (daan944) [reply client/coral-framework/services/i18n.js]: Yes! I've just commited the change.

**Review by kgardnr — APPROVED** [review]

**Pull request merged**